ast/state_variable.go

```diff
@@ -22,6 +22,7 @@ type StateVariableDeclaration struct {
 	Visibility      ast_pb.Visibility      `json:"visibility"`        // Visibility of the state variable declaration
 	StorageLocation ast_pb.StorageLocation `json:"storage_location"`  // Storage location of the state variable declaration
 	StateMutability ast_pb.Mutability      `json:"mutability"`        // State mutability of the state variable declaration
+	Override        bool                   `json:"is_override"`       // Indicates if the state variable is an override
```

Comment:
Introduced an Override for StateVariableDeclaration, eg ` address public override owner;`